src/__tests__/ConfirmationDialog.test.tsx

```diff
@@ -62,10 +63,12 @@ const ShorthandHookFromActionMenu = () => {
       <SSRProvider>
         <BaseStyles>
           <Box display="flex" flexDirection="column" alignItems="flex-start">
-            <ActionMenu
-              renderAnchor={props => <Button {...props}>{text}</Button>}
-              items={[{text: 'Show dialog', onAction: onButtonClick}]}
-            />
+            <ActionMenu>
+              <Button>{text}</Button>
+              <Button ref={buttonRef} onClick={onButtonClick}>
+                Show dialog
+              </Button>
```

Comment:
```suggestion

              <ActionMenu.Button>{text}</ActionMenu.Button>
              <ActionMenu.Overlay>
                <ActionList>
                  <ActionList.Item onSelect={onButtonClick}>Show dialog</ActionList.Item>
                </ActionList>
              </ActionMenu.Overlay>

```

Reply:
The test might be succeeding but I added a suggestion to encourage the correct usage of `ActionList` according to the [docs](https://primer.style/react/ActionMenu/) 

We need to make sure to import `ActionList` as well. 

```
import {ActionList} from '../ActionList'
```

Let me know if you have any questions about this!

Reply:
@ItzaMi Sorry, I missed your commit! 

Seems like the only thing we are now missing is importing the `ActionList` to `src/__tests__/ConfirmationDialog.test.tsx`

```
import {ActionList} from '../ActionList'`
```